packages/exiflooter/PKGBUILD

```
@@ -0,0 +1,53 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=exiflooter
+pkgver=33.a92e697
+pkgrel=1
+pkgdesc='Find geolocation on all image urls and directories also integrates with OpenStreetMap.'
+arch=('any')
+groups=('blackarch' 'blackarch-forensic')
+url='https://github.com/aydinnyunus/exifLooter'
+license=('Apache-2.0')
```

```suggestion
license=('Apache')
```

cf. https://wiki.archlinux.org/title/PKGBUILD#license